postgrest.cabal

```diff
@@ -24,6 +24,7 @@ Flag CI
 executable postgrest
   main-is:             PostgREST/Main.hs
   default-extensions:  OverloadedStrings, ScopedTypeVariables, QuasiQuotes, LambdaCase
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
```

Comment:
days of banging the head against the wall, fixed by a compile option :) welcome to concurrency. I am really glad we are in the haskell world right now and not in C semaphores and mutexes :D